gradio/components/audio.py

```diff
@@ -142,7 +145,12 @@ def __init__(
             if show_share_button is None
             else show_share_button
         )
-        self.waveform_options = waveform_options
+
+        self.waveform_options = (
```

Comment:
looks good!

Reply:
Actually we may also need to handle the case when `waveform_options=None`. Right now, if I don't pass in any `waveform_options`, no waveform is shown while recording, instead of the default `WaveformOptions()`